src/host/tcp.h

```diff
@@ -594,6 +612,12 @@ namespace asynchost
         return;
       }
 
+      if (sz == UV_ENOBUFS)
+      {
+        LOG_DEBUG_FMT("TCP on_read reached allocation quota");
```

Comment:
Minor: `LOG_TRACE_FMT()`?

Reply:
I think this should be rare enough that `DEBUG` is fine. There's already a `TRACE` in `on_alloc` with more info. I think if you're in a `DEBUG` but not `TRACE` run, you want to recognise any unusual paths which are taken, which this is.